arangod/Cluster/Maintenance.cpp

```diff
@@ -611,21 +649,38 @@ arangodb::Result arangodb::maintenance::executePlan(VPackSlice const& plan,
     // open ACTIONS
     TRI_ASSERT(report.isOpenObject());
     report.add(ACTIONS, VPackValue(VPackValueType::Array));
-  
-    // enact all
-    for (auto& action : actions) {
-      LOG_TOPIC("8513c", DEBUG, Logger::MAINTENANCE)
-          << "adding action " << action.get() << " to feature ";
+  }
+
+  // enact all
+  for (auto& action : actions) {
+    LOG_TOPIC("8513c", DEBUG, Logger::MAINTENANCE)
+        << "adding action " << action.get() << " to feature ";
+    if (debugActions) {
       VPackObjectBuilder b(&report);
       action->toVelocyPack(report);
+    }
+    if (!action->isForced()) {
```

Comment:
Should the condition be reversed, e.g.
```suggestion
    if (action->isForced()) {
```